docker/ceph/start-ceph.sh

```diff
@@ -5,7 +5,7 @@ set -e
 source /docker/set-start-env.sh
 
 # Build frontend:
-if [[ "$FRONTEND_BUILD_REQUIRED" == 1 ]]; then
+if [[ "$FRONTEND_BUILD_REQUIRED" == 11 ]]; then
```

Comment:
I assume this change is not intentional.

Reply:
oops